### PR TITLE
Create printer-anicubic-kobra-neo-2022.cfg

### PR DESCRIPTION
**Pull request by Michel-lev**

### CODE REVIEW

**Comment by bistory** [comment]
Thanks ;-)
Can you rename the file and fix the typo ? You wrote "anicubic" instead of anycubic :-)
Can you also move variables out of save data section (and remove them), so they are hard written in the configuration.
Except z_offset for probe, you have to leave it to 0, it's the user responsability to put the correct value there.
Please remove the include for macros folder, it doesn't exist in the cheapad configuration
Also, can you add temperature sensors for rpi and mcu, and add exclude_objects + gcode_arcs like on all other configurations ?

**Comment by Michel-lev** [comment]
bonjour modification faite

File: Anycubic/printer-anycubic-kobra-neo-2022.cfg [hunk 1]
```diff
@@ -116,12 +113,15 @@ sensor_type: EPCOS 100K B57560G104F
 sensor_pin: PC1
 min_temp: 0
 max_temp: 120
+control = pid
+pid_kp = 59.770
+pid_ki = 0.960
+pid_kd = 930.174
```
Comment (bistory) [inline Anycubic/printer-anycubic-kobra-neo-2022.cfg]: Utilise la syntaxe classique avec les ":" au lieu du "=" :-)

**Review by bistory — CHANGES_REQUESTED** [review]
Il reste l'include des macros à enlever aussi ;-)

**Comment by Michel-lev** [comment]
fait

**Comment by 1coderookie** [comment]
Just fyi guys: seems like an old version of my example printer.cfg for the Neo has been used by @Michel-lev. There are wrong settings in the file which this PR is about.  
E.g.:   
1. This setting is wrong:  
```
[probe]
...
y_offset: -4
```  
The y-offset must be a positive 4, not a negative.   

2. This `homing_positive_dir: false` is obsolete as it's set false per default due to the general config here and therefore an already active setting shouldn't be declared again (regarding the official Klipper docs).  

@bistory You can find a correct file here:  https://github.com/1coderookie/Klipper4KobraGoNeo/blob/main/cfg/NEO_1coderookie_printer.cfg   
Users would have to check and adjust the settings anyway though, e.g. it might be that the head doesn't go low enough due to  
```
[stepper_z]
...
position_min: -2
```
which might be a value too small as the position of the lever which triggers the z-limit switch doesn't seem to be consistent across the models (and I also use a slightly modified lever at my machine).   

Besides that, using the probe as a virtual endstop can be done, but it's imho not advisable to do so due to the risk of a failing probe (we had those issues reported which were mostly caused by broken wirings of the probe) and therefore the risk of driving the nozzle into the bed.  

Just as an information for you @bistory as it might be helpful for users and your project to mention this repo for users who have a Kobra Go / Kobra Neo:  https://github.com/1coderookie/Klipper4KobraGoNeo  - there they can also find a precompiled klipper.bin.  

Just in case you're interested in these specific models (Kobra Go & Kobra Neo), here you'll find a specific information site I set up for them: https://1coderookie.github.io/KobraGoNeoInsights/  
Chapter about Klipper at these machines: https://1coderookie.github.io/KobraGoNeoInsights/firmware/fw_klipper/  

Hope it might be helpful somehow.. ;)

**Comment by bistory** [comment]
Thanks, I added the config @1coderookie !

**Comment by 1coderookie** [comment]
Welcome! :)

**Pull request closed without merging**